components/organisms/Drawer.js

```diff
@@ -0,0 +1,82 @@
+import PropTypes from "prop-types";
+import React, { useState } from "react";
+import { useTranslation } from "next-i18next";
+import FocusTrap from "focus-trap-react";
+import { ActionButton } from "../atoms/ActionButton";
+
+export const Drawer = ({ children }) => {
+  const [isOpen, setisOpen] = useState(false);
+  const { t } = useTranslation("common");
+
+  const toggleDrawer = () => {
+    setisOpen(!isOpen);
+    const drawerEl = document.getElementById("drawer");
+    const bodyEl = document.body.style;
```

Comment:
drawerEl is an element and bodyEl is the style. These can be made more consistent by both being elements or style since they are used in the same way